apiserver/facades/client/modelgeneration/modelgeneration.go

```diff
@@ -238,7 +250,8 @@ func (api *API) AbortBranch(arg params.BranchArg) (params.ErrorResult, error) {
 // including units on the branch and the configuration disjoint with the
 // master generation.
 // An error is returned if no in-flight branch matching in input is found.
-func (api *API) BranchInfo(args params.BranchInfoArgs) (params.GenerationResults, error) {
+func (api *API) BranchInfo(
+	args params.BranchInfoArgs) (params.GenerationResults, error) {
 	result := params.GenerationResults{}
```

Comment:
Something I think we *should* do, is rename these:
- `GenerationResults` to `BranchResults`.
- `GenerationCommitResult` to `GenerationResult`.
- `GenerationcommitResults` to `GenerationResults`.